### PR TITLE
Store Storyblok cache version in environment variable

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

- During the build, read latest Storyblok cache version and store in env variable

- Remove logic related to reading and writing cache version to Edge Config

## Justify why they are needed

- Edge config is eventually consistent so it's not ideal to store the cache version there if we need to read it again right after

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2745** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2745" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2745?utm_source=stack-comment).

File: bin/vercel-ignore-step.sh [hunk 1]
```diff
@@ -13,4 +13,4 @@ if [ "$VERCEL_GIT_COMMIT_SHA" = "$VERCEL_GIT_PREVIOUS_SHA" ]; then
   exit 1
 fi
 
-yarn dlx turbo-ignore
+npx turbo-ignore
```
Comment (alebedev) [inline bin/vercel-ignore-step.sh:16]: Why npx?

**Review by alebedev — APPROVED** [review]
Looks good

File: bin/vercel-ignore-step.sh (hunk 1 above)
Comment (robinandeer) [reply bin/vercel-ignore-step.sh:16]: it's what vercel recommended -- however, I don't think it fixed the problems I was seeing so let's revert

**Pull request merged**